src/script/standard.h

```diff
@@ -163,10 +163,11 @@ enum KeyType {
     PKHashKeyType = 1 << 0,
     ScriptHashKeyType = 1 << 1,
     WPKHashKeyType = 1 << 2,
-    EthHashKey = 1 << 3,
+    EthHashKeyType = 1 << 3,
```

Comment:
We're using ERC55 terminology now.

Reply:
We can rename the others as well to use the same terminology (in a separate PR), but let's move that direction, than this direction :)

Reply:
Got it. We can do the fix for this in a separate PR.